subsys/net/l2/ethernet/gptp/gptp_messages.c

```diff
@@ -188,6 +188,7 @@ struct net_pkt *gptp_prepare_sync(int port)
 
 	pkt = setup_gptp_frame(iface);
 	if (!pkt) {
+		NET_DBG("Cannot get gPTP frame");
```

Comment:
Why not generating this dbg message in setup_gptp_frame directly?

Reply:
Then we would not know which packet generation function failed and what packet was not sent.